dao/database.js

```diff
@@ -84,94 +86,8 @@ const promisedDb = {
 
 const readyDeferred = deferred();
 
-// NOTE: members_reminders(member_id) has no ON DELETE CASCADE because we need
-// to manually cascade to reminders if there is no more recipients associated to
-// a reminder.
-const createStatements =
-`
-  CREATE TABLE IF NOT EXISTS families
-  (
-    name TEXT PRIMARY KEY
-  );
-
-  CREATE TABLE IF NOT EXISTS members
-  (
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
-    family TEXT NOT NULL
-      REFERENCES families(name)
-      ON UPDATE CASCADE
-      ON DELETE CASCADE
-      DEFERRABLE INITIALLY DEFERRED,
-    name TEXT NOT NULL,
-    UNIQUE (family, name)
-  );
-
-  CREATE TABLE IF NOT EXISTS members_families
-  (
-    member_id INTEGER NOT NULL
-      REFERENCES members(id)
-      ON UPDATE CASCADE
-      ON DELETE CASCADE
-      DEFERRABLE INITIALLY DEFERRED,
-    family_name TEXT NOT NULL
-      REFERENCES families(name)
-      ON UPDATE CASCADE
-      ON DELETE CASCADE
-      DEFERRABLE INITIALLY DEFERRED,
-    accepted BOOLEAN,
-    PRIMARY KEY (member_id, family_name)
-  );
-  CREATE INDEX IF NOT EXISTS members_families_family
-    ON members_families (family_name);
-
-  CREATE TABLE IF NOT EXISTS reminders
-  (
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
-    action TEXT,
-    created INTEGER NOT NULL, -- in milliseconds
-    due INTEGER NOT NULL, -- in milliseconds
-    status TEXT DEFAULT 'waiting'
-  );
-  CREATE INDEX IF NOT EXISTS reminders_due
-    ON reminders(due);
-  CREATE INDEX IF NOT EXISTS reminders_status
-    ON reminders(status);
-
-  CREATE TABLE IF NOT EXISTS members_reminders
-  (
-    member_id INTEGER NOT NULL
-      REFERENCES members(id)
-      ON UPDATE CASCADE
-      DEFERRABLE INITIALLY DEFERRED,
-    reminder_id INTEGER NOT NULL
-      REFERENCES reminders(id)
-      ON UPDATE CASCADE
-      ON DELETE CASCADE
-      DEFERRABLE INITIALLY DEFERRED,
-    PRIMARY KEY (member_id, reminder_id)
-  );
-  CREATE INDEX IF NOT EXISTS members_reminders_reminder_id
-    ON members_reminders(reminder_id);
-
-  CREATE TABLE IF NOT EXISTS subscriptions
-  (
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
-    member_id INTEGER NOT NULL
-      REFERENCES members(id)
-      ON UPDATE CASCADE
-      ON DELETE CASCADE
-      DEFERRABLE INITIALLY DEFERRED,
-    title TEXT,
-    endpoint TEXT UNIQUE,
-    p256dh TEXT,
-    auth TEXT
-  );
-  CREATE INDEX IF NOT EXISTS subscriptions_member_id
-    ON subscriptions(member_id);
-`;
-
-function init(profileDir) {
-  const dbPath = path.join(profileDir, 'reminders.db');
+function init(profileDir, name) {
```

Comment:
it's not needed to pass a name: the tests are run in another profile directory :)